### PR TITLE
Solved problem

### PR DESCRIPTION
**Pull request by annadolan**

### CODE REVIEW

**Comment by jcasimir** [comment]
👍

File: well-formed-strings-test.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    v = StringValidator.new
+    assert v.validate("([{}[]])"), true
+  end
+
+  def test_one_off
+    v = StringValidator.new
+    assert v.validate("()["), false
+  end
+
+  def test_offset
```
Comment (mikedao) [inline well-formed-strings-test.rb:21]: I appreciate the fact that you included sad path testing here. 

You were the first to do so. 

Brava!

File: well-formed-strings.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+require 'pry'
+class StringValidator
+  def validate(input)
+    result = false
+    match = {"}" => "{", "]" => "[", ")" => "("}
```
Comment (mikedao) [inline well-formed-strings.rb:5]: Can this be pulled out in any manner? Can it change? Does it change? Is it reusable?

File: well-formed-strings.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    array = input.split(//)
+    count = 0
+    keep = []
+    while count <= array.length
```
Comment (mikedao) [inline well-formed-strings.rb:9]: Is there an enumerable that could also perform this task?

**Review by mikedao — APPROVED** [review]

**Pull request closed without merging**